### ISSUE TITLE
can't connect

### ISSUE THREAD

**Issue by ImperialCrise** [adw0rd/instagrapi#252]
i must spam like 4 times the script to successfully connect to instagram.

the error 👍 
```
`
Traceback (most recent call last):
  File "pontinsta.py", line 94, in <module>
    upload_post()
  File "pontinsta.py", line 55, in upload_post
    bot.login(username = username, password = password)
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/auth.py", line 382, in login
    self.login_flow()
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/auth.py", line 159, in login_flow
    check_flow.append(self.get_timeline_feed([chance and "is_pull_to_refresh"]))
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/auth.py", line 215, in get_timeline_feed
    return self.private_request(
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/private.py", line 356, in private_request
    raise e
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/private.py", line 343, in private_request
    self._send_private_request(endpoint, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/instagrapi/mixins/private.py", line 244, in _send_private_request
    raise LoginRequired(response=e.response, **last_json)
instagrapi.exceptions.LoginRequired: login_required
`
```

**Comment by dheeraj009joshi** [comment]
I have the same issue i had tried with 15 accounts but the err was same

**Comment by MarquesH246** [comment]
Hi, 
I'm having the same issue. Can someone help ?
thank you.

**Comment by adw0rd** [comment]
Fixed in instagrapi 1.8.2

**Comment by MeetK208** [comment]
I have currently facing this error from past 3 days on latest version.I am using user_info-by_username without login for public user access but it will gives error this every time i had tried with 30 accounts but the err was same.
